src/main/java/net/fabricmc/loader/game/MinecraftGameProvider.java

```diff
@@ -158,6 +159,31 @@ public void acceptArguments(String... argStrs) {
 		FabricLauncherBase.processArgumentMap(arguments, envType);
 	}
 
+	@Override
+	public String[] getLaunchArguments(boolean sanitize) {
+		if (arguments != null) {
+			List<String> list = new ArrayList<>(Arrays.asList(arguments.toArray()));
+			if (sanitize) {
+				int remove = 0;
+				Iterator<String> iterator = list.iterator();
+				while (iterator.hasNext()) {
+					String next = iterator.next();
+					if ("--accessToken".equals(next)) {
```

Comment:
Hmm, I'd like to ask is the accessToken the only compromising info we want to sanitize?

Multimc for example actually omits your username, uuid and user type in logs alongside your accessToken like this:
`--username  --version MultiMC5 --gameDir /home/i509vcb/.local/share/multimc/instances/1.16.2 - FABRIC/.minecraft --assetsDir /home/i509vcb/.local/share/multimc/assets --assetIndex 1.16 --uuid --accessToken  --userType  --versionType release`

Reply:
I wouldn't consider the username or UUID sensitive information personally. The UUID is given out to anyone you are on a server with, as well as stored in plain text on your hard drive, and the username is already logged on startup and can be changed. What does user type do?

Reply:
MC logs/dumps the other information quite happily, I think of this sanitizing as a security feature, not a privacy one.